File: c program/other code/prime number.c
#include <stdio.h>
int main()
{
    int i, num,count=0;
    printf("Enter the number");
    scanf("%d",&num);
    for(i=2; i<=num; i++)

        if(num % i ==0)
        {
            printf("Not Prime Number");
            count++;
            break;
        }
    
    else
    {
        printf("Number is Prime");
    }
    return 0;
}

File: Java Programming/Day2/src/Task1/primeNumber.java
package Task1;

import java.util.Scanner;

public class primeNumber 
{
    static boolean isPrime(int n)
    {
        if (n <= 1)
            return false;
        for (int i = 2; i < n; i++)
            if (n % i == 0)
                return false;
    
        return true;
    }

    public static void main(String args[])
    {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter Number: ");
        
        int n = input.nextInt();
        
        if (isPrime(n))  
            System.out.println(n + " is prime");
        else
            System.out.println(n + " is not prime");
    }
}
